SEED-platform/seed
# Add option to export meter data in GeoJSON

#### Any background context you want to provide?
Exporting Meter data isn't readily available. One can export from the "meters" tables within a single property.

#### What's this PR do?
Since meters and meter readings are relational, adding this export functionality to a flat CSV file doesn't make sense. We could eventually add to XLSX (similar to ESPM). This PR adds the data to a geojson record.

* Adds meters and meter readings to the property obj in geojson
* Adds option to exclude the export in the modal (the default is to exclude)

#### How should this be manually tested?
* go to an org with properties that have meters
* select a few properties
* check the "export meter readings" and then "export geojson"
* verify that the exported geojson file has the meter data.

#### What are the relevant tickets?
further address #2946

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/1907354/225496048-4a62903e-42e8-4972-ba76-738f304a5b87.png)

![image](https://user-images.githubusercontent.com/1907354/225496137-5ae93480-7a7c-4a60-8f94-1f73c980d416.png)
